frontend/javascripts/oxalis/model/accessors/dataset_layer_transformation_accessor.ts

```diff
@@ -239,7 +243,7 @@ export function getTransformsForLayer(
   );
 }
 
-export function isIdentityTransform(transform: Transform) {
+function isIdentityTransform(transform: Transform) {
```

Comment:
how about renaming this to `equalsIdentityTransform` to make it clearer that it checks for equality (because identity can now always be checked with ===).